### PR TITLE
two genes one plate

### PR DESCRIPTION
**Pull request by MartinFXP**
I removed duplicated filenames in the tablefile object.
Can that lead to any problems?

### CODE REVIEW

**Comment by dirmeier** [comment]
Thanks Martin, I need to look into this. The fix introduced Travis to fail. `reindex` and `reindex_axis` do the same, on;ly that the latter is deprecated.

**Comment by MartinFXP** [comment]
I think the previous fix 2ec7c80 for issue #20, was only considering, when you get several identical entries for one unique query. E.g. you query "mock" control and you'd get several times a single plate, because a single plate contains several "mock" controls. However, for distinct queries, as in "gene1,gene2", this will not solve the issue, since the queries for "gene1" and "gene2" might give by themselves might be unique or will be made unique by the fix 2ec7c80, but can still have and overlap. At least that is the best explanation I can come up with.

**Comment by dirmeier** [comment]
Hey Martin, 
I think I identified the error and it was further upstream already. So while contribution fixes the error, it does not fix the bug. If you could add the reviewes I made, Ill merge.

File: rnaiutilities/table_file_set.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class TableFileSet:
     def __init__(self, key, query_result, features, **kwargs):
         self._table_file_set_classifier = key
         f = [x[-1].replace("_meta.tsv", "") for x in query_result]
-        self.file_names = [el + "_data.tsv" for el in f]
+        self.file_names = list(set([el + "_data.tsv" for el in f]))
```
Comment (dirmeier) [inline rnaiutilities/table_file_set.py]: Replace `list(set(...))` with `unique`please.

**Pull request merged**